src/daemon/bitcoind/interface.rs

```diff
@@ -889,6 +890,19 @@ impl BitcoinD {
             Err(e) => Err(e),
         }
     }
+
+    /// Check whether a transaction is part of the wallet, and not stuck (as in is confirmed or
+    /// part of the mempool).
+    pub fn is_current(&self, txid: &Txid) -> Result<bool, BitcoindError> {
```

Comment:
I think the naming here should be improved a bit, because it's not trivial for me to understand what this function does
So it returns true if it's in mempool or in the blockchain, false otherwise? `in_wallet` maybe?

Reply:
Or is `current` a standard term that I had never heard? :)

Reply:
No, i don't think it is ;) Suggestions are welcome, but i want to express "wallet tx that is either in mempool or block chain". I wanted to call it is_valid at first but it clashes with PSBTs validity so imo it's not clear. So i though about `is_currently_valid` (as in is valid according to the current tip), hence `is_current` :D

Reply:
I have no better ideas :( Let's leave it as it is, then we can always refactor